### ISSUE TITLE
[Adaptive Cards> Schema Explorer> Action.Execute]: Placeholder text is not defined for 'Please enter your name below' edit field.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7920]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25197.1000)
Browser: Edge Dev (107.0.1379.1)
URL: https://adaptivecards.io/designer/

**Repro steps:**

1.  Open the above URL in edge dev browser and login with v-.
2.  Navigate to the 'Schema Explorer' tab through tab key and activate it.
3.  Navigate to the 'Action.execute' button through tab key.
4.  Navigate to the 'Please enter your name below' edit field through tab key.
5.  Observe the issue.

Actual Result:
On Schema Explorer tab, Placeholder text is not defined for 'Please enter your name below' edit field.

Expected Result:
On Schema Explorer tab, Placeholder text should be properly defined for 'Please enter your name below' edit field. Such as **'Enter 'First name' and 'Last name'.**

**Notes:**
Same issue repro with 'What is your name' and 'What is your last name' edit field under avction.execute button.

**User Impact:**
People with cognitive disability will get impacted as user will not be able to understand the purpose of the edit field and will not be able to navigate or track the content properly.

Usable Reference link:
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/doc2.aspx?sourcedoc=%7B64DD7C5C-AA62-4D53-886E-BCAEE5341C01%7D&file=Guideline%20101%20Recommended%20Assistive%20Technology%20Tools%20for%20MAS%20Testing%20of%20Microsoft%20Digital%20Properties_v3.docx&action=default&mobileredirect=true&cid=e5999b53-8463-45ff-86b3-086ee5e5c931&isSPOFile=1&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cv-mohittyagi%40microsoft.com&wdOrigin=TEAMS-ELECTRON.p2p.bim&wdExp=TEAMS-CONTROL&wdhostclicktime=1658296286674

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="960" alt="Usable~ Placeholder text is not defined foe please enter your nazme below" src="https://user-images.githubusercontent.com/104483005/191739858-f7a1ed92-8ccc-42e0-93ed-f2ef6e2c963f.png">

### Card JSON

```JSON
NA
```
